API_doc/LibConfig.hpp

```diff
@@ -0,0 +1,65 @@
+//
+// Must be included to use the LibConfig
+//
+//
+
+#pragma once
+
+# include <cstddef>
+
+/*
+ * Base class for all operations
+ */
+class LibConfig
```

Comment:
A namespace would be more appropriate.

Reply:
I do that to keep all functions in a class. But if it's not a problem to have functions out of class or if you have another organization in mind, it's okay for me to write *LibConfig* as a namespace.

Reply:
A class only makes sens if you need members, which we don't have in this case.